### PR TITLE
[RFR][1LP] enable the venv to be build on py3k using the template files

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
this also makes ´miq shell` start without exceptions

### CODE REVIEW

File: cfme/scripting/quickstart.py [hunk 1]
```diff
@@ -280,21 +286,22 @@ def install_requirements(venv_path, quiet=False):
...
+    if last_hash == current_hash and not PY3:
         print("INFO: skipping requirement installation as frozen ones didn't change")
         print("      to enforce please invoke pip manually")
         return
 
     elif last_hash is not None:
         current_packages = pip_json_list(venv_path)
         print("INFO:", REQUIREMENT_FILE, 'changed, updating virtualenv')
-
+    elif PY3:
```
Comment (psav) [inline cfme/scripting/quickstart.py]: **REQUIRED:** Does this mean that at every invocation of ```miq shell``` we will do an install?

File: fixtures/browser.py [hunk 2]
```diff
@@ -35,11 +35,10 @@ def pytest_exception_interact(node, call, report):
     from fixtures.pytest_store import store
     from six.moves.http_client import BadStatusLine
     from socket import error
-    import urllib2
 
     val = safe_string(call.excinfo.value.message).decode('utf-8', 'ignore')
-
-    if isinstance(call.excinfo.value, (urllib2.URLError, BadStatusLine, error)):
+    from six.moves.urllib_error import URLError
```
Comment (psav) [inline fixtures/browser.py]: **REQUIRED:** Why not at top of file?

File: widgetastic_manageiq/__init__.py [hunk 3]
```diff
@@ -868,7 +868,7 @@ def click_at(self, item_name):
 
         return self.list(item_name).list_click()
 
-    def is_displayed(self, (item_name)):
```
Comment (psav) [inline widgetastic_manageiq/__init__.py]: **REQUIRED:** Can you explain this one a little please?

File: cfme/scripting/quickstart.py (hunk 1 above)
Comment (RonnyPfannschmidt) [reply cfme/scripting/quickstart.py]: for now it does - fixing freezing will elevate it

File: fixtures/browser.py (hunk 2 above)
Comment (RonnyPfannschmidt) [reply fixtures/browser.py]: this was just the site

File: widgetastic_manageiq/__init__.py (hunk 3 above)
Comment (RonnyPfannschmidt) [reply widgetastic_manageiq/__init__.py]: the syntax is equivalent on python2 as it allowed tuple unpacking in function arguments,
its a syntaxerror on python3

**Pull request merged**